### PR TITLE
Creating Job_Post_UI

### PR DESCRIPTION
**Pull request by lerabo**
localhost_3000_job-post (1)

### CODE REVIEW

File: src/components/jobPost/JobPost.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+		descriptionError,
+		fromHourRate,
+		fromHourRateError,
+		onFromHourRateChange,
+		toHourRate,
+		toHourRateError,
+		onToHourRateChange,
+		onSubmitErrors,
+		onSubmit,
+	} = useOnDataChange();
```
Comment (lyudad) [inline src/components/jobPost/JobPost.tsx]: hmm I think we can use react-hook-form

File: src/components/jobPost/components/RadioBox.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+interface Props {
+	onChange: (event: RadioChangeEvent) => void;
+	value: string;
+}
+
+const RadioBox = (props: Props) => {
+	return (
+		<Radio.Group onChange={props.onChange} value={props.value}>
+			<Space direction="vertical">
+				<Radio value={'0-1 month'}>{`${t('JobPostPage.shortMonthDuration')}`}</Radio>
```
Comment (lyudad) [inline src/components/jobPost/components/RadioBox.tsx]: all texts should be in text json

File: src/components/jobPost/JobPost.tsx (hunk 1 above)
Comment (lerabo) [reply src/components/jobPost/JobPost.tsx]: Used

File: src/components/jobPost/components/RadioBox.tsx (hunk 2 above)
Comment (lerabo) [reply src/components/jobPost/components/RadioBox.tsx]: Fixed

File: src/components/jobPost/JobPost.styles.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	LIGHTBLUE_COLOR,
+	RED_COLOR,
+	WHITE_COLOR,
+} from 'constants/colors';
+import { LARGE_FONT_SIZE } from 'constants/fonts';
+import styled from 'styled-components';
+
+export const Container = styled.div`
+	padding: 1% 8% 1% 8%;
+	font-size: 20px;
```
Comment (lyudad) [inline src/components/jobPost/JobPost.styles.ts]: move to constants

File: src/components/jobPost/JobPost.tsx [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+	description: string;
+	duration: string;
+};
+
+const JobPost = () => {
+	const { t } = useTranslation();
+
+	const { skillsOptions, onSkillsChange, filteredSkills } = useOnDataChange();
+
+	const validationSchema = Yup.object().shape({
```
Comment (lyudad) [inline src/components/jobPost/JobPost.tsx]: move to separate file

File: src/components/jobPost/JobPost.tsx [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+		title: Yup.string()
+			.required(`${t('JobPostPage.fieldIsRequired')}`)
+			.min(6, `${t('JobPostPage.lengthWarning')}`)
```
Comment (lyudad) [inline src/components/jobPost/JobPost.tsx]: we dont translate strings that we use for conditions and for validation so on

File: src/components/jobPost/JobPost.tsx [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+	type NotificationType = 'success' | 'error';
+	const openNotificationWithIcon = (type: NotificationType) => {
+		notification[type]({
+			message: type === 'success' && `${t('JobPostPage.success')}`,
+			description: type === 'success' && `${t('JobPostPage.dataHasBeenSaved')}`,
+		});
+	};
+
+	const onSubmit = (data: JobSubmitForm) => {
+		console.log(JSON.stringify(data, null, 2));
```
Comment (lyudad) [inline src/components/jobPost/JobPost.tsx]: delete console

**Pull request closed without merging**